### ISSUE TITLE
Add  sending answers to Virtual caches

### ISSUE THREAD

**Issue by okainov** [cgeo/cgeo#7181]
Almost always for Virtual (or Earth) cache it's required to send answer to the cache owner \ to geocaching website.

Specifically, on geocaching.su there is automated system for checking virtual caches answers. So it would be useful to have possibility to send it directly from c:geo.

Currently I have two ideas in mind:

- Develop an extension for standard logging page in the same way as it's done for voting. 
+ No need to develop new activity
+ Log and answer will go together, less possibility to forget to send the answer afterwards
\- May overload the page even more, just saw the "full" variant and adding extra text field to this one may be a bit extra
![image](https://user-images.githubusercontent.com/918446/48280086-db26bb80-e452-11e8-8e6e-11a8f0b74f0a.png)

- Do it via new menu bullet "send answer"
+ Will not conflict\overload existing UI
\- Need to develop SendAnswerActivity
\- May be forgotten during the log.

I would ask dev team for feedback on this, do you think it will fit in c:geo, and if yes, which option do you prefer?

**Comment by SammysHP** [comment]
Maybe you can reuse the log password function used for OpenCaching?

**Comment by Lineflyer** [comment]
Yes, if it comes to a password it would be preferable to resuse the password field for OC virtuals, which is already part of the logging view. 
However for OC this is not used to contact anyone but to automatically check the logging permission.

**Comment by okainov** [comment]
>Maybe you can reuse the log password function used for OpenCaching?

Just checked it - seems the behavior a bit hardcoded for specific case, when the password is really required to log the cache. However in my case, I consider this as optional.

Also, if the translation strings translate this as "log password", it will be really confusing, because we don't have "password" concept on SU, it's exactly "answer"
